MultiTarget/PackageReferences/WinAppSdk.props

```diff
@@ -1,6 +1,9 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.5.240311000" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240531000-experimental1" />
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.1.0-prerelease.240602.1" />
```

Comment:
This should have `PrivateAssets="all"` for all Toolkit libraries (anything that's not an app).

Reply:
Resolved in 49cb231c9e26fce7627e6491890774b71a9552db

Reply:
Reverting to work around [this issue](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/195#issuecomment-2294395134) temporarily.